packages/flutter/lib/src/widgets/image.dart

```diff
@@ -188,12 +188,6 @@ class _ImageState extends State<Image> {
   ImageInfo _imageInfo;
 
   @override
-  void initState() {
-    super.initState();
-    _resolveImage();
```

Comment:
this one is wrong. We do want to do this in initState. It's safe to do so because dependenciesChanged takes care of the reevaluation when the inherited values change.